src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -25,10 +25,13 @@
   []
   (let [selected-tab (reagent/atom first-tab-id)]
     (fn []
-      (let [{:keys [name color formatted-balance
-                    watch-only?]} (rf/sub [:wallet/current-viewing-account])
-            customization-color   (rf/sub [:profile/customization-color])]
-        (rn/use-unmount #(rf/dispatch [:wallet/close-account-page]))
+      (let [{:keys [name color formatted-balance watch-only?
+                    address]}   (rf/sub [:wallet/current-viewing-account])
+            customization-color (rf/sub [:profile/customization-color])]
+
+        (rn/use-mount
+         #(rf/dispatch [:wallet/fetch-activities-for-current-account address]))
```

Comment:
Remove the `use-unmount` to navigate back because it'll make the app hard to develop.

Added the event to fetch activities for the current account